exporter/jaeger/jaeger.go

```diff
@@ -39,6 +39,11 @@ type Options struct {
 	// For example, http://localhost:14268.
 	Endpoint string
 
+	// EndpointPath is the url path where spans are sent.
+	// If this is not set, it will send to "/api/traces" as the default
+	// For example, /api/traces
+	EndpointPath string
```

Comment:
After looking at the rest of the options, I think it is a little confusing that there is both `Endpoint` and `EndpointPath`. The question that occurred to me while looking at the new API is: why not include the path in the Endpoint option instead of having a separate `EndpointPath` option? 

Because of backwards compatibility, we cannot change the meaning of `Endpoint`. So instead I would like to recommend we introduce a new option: `CollectorEndpoint string`. If set, this will be the full URL (including the path) to the collector. We then deprecate `Endpoint` in favor of `CollectorEndpoint`. 

`CollectorEndpoint` is also more descriptive fits better with `AgentEndpoint`.

What do you think?

Reply:
I agree that `CollectorEndpoint` seems a lot clearer than `Endpoint` + `EndpointPath`. I will make this change

Do you think it's a good idea to add a log message now that `Endpoint` is deprecated? Or should that be a change for later?